Permit extended keyboard arrow keys to be used for keyboard joystick

ejaquay opened the pull request:
Note: While keys are mapped to joystick(s) the keypad 4,8,6,
and 2 keys will also be mapped as arrow keys.  Keys mapped to the
joystick(s) will no longer have their normal function.

BGPierce commented:
Lee, does it compile and run on your machine?
The current build was compiling and running (per the release), but I've yet to try it on VS2017, so that may be the issue. I should have VS2017 installed by early next week and I'll try the current revision before merging yours just to see if I have the same issues. If so, then the cause is definately changes made in VS2017.

BGPierce commented:
Wes just emailed me and said his new revision already has those changes, but I will add them to this revision after I test them.

 

 

Bill Pierce
"Charlie stole the handle, and the train it won't stop going, no way to slow down!" - Ian Anderson - Jethro Tull

My Music from the Tandy/Radio Shack Color Computer 2 & 3
https://sites.google.com/site/dabarnstudio/
Co-Contributor, Co-Editor for CocoPedia
http://www.cocopedia.com/wiki/index.php/Main_Page

E-Mail: ooogalapasooo@aol.com

 

 

-----Original Message-----
From: Abathur8bit <notifications@github.com>
To: VCCE/VCC <VCC@noreply.github.com>
Cc: Bill Pierce <ooogalapasooo@aol.com>; Comment <comment@noreply.github.com>
Sent: Fri, Mar 9, 2018 8:34 pm
Subject: Re: [VCCE/VCC] Fix for 6809 TFR W,Y crashing VCC (#80)



Yup, compiles and runs with the changes I checked in.
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or mute the thread.

Commit:
Permit extended keyboard arrow keys to be used for keyboard joystick

Note: While keys are mapped to joystick(s) the keypad 4,8,6,
and 2 keys will also be mapped as arrow keys.  Keys mapped to the
joystick(s) will no longer have their normal function.
